### ISSUE TITLE
UTF-8 Encoding

### ISSUE THREAD

**Issue by lgignoud** [tuarua/WebViewANE#125]
I have an issue with string encoding, when using webView.callJavascriptFunction(): I can't get the strings on the javascript side to be encoded in UTF-8.

The problem can be reproduced in the example function *StarlingRoot.onAsJsAsBtn()*, by passing a string with an accent:
```javascript
webView.callJavascriptFunction("as_to_js", asToJsCallback, 1, "é", 77);
```
When show in devTools, the "é" become "Ã©".

Am I missing some settings ?

**Comment by tuarua** [comment]
Is this on Windows ?

**Comment by lgignoud** [comment]
Hi Tuarua,Yes, Windows 7.Le 1 mars 2018 à 18:38, Tua Rua <notifications@github.com> a écrit :  Is this on Windows ?—You are receiving this because you authored the thread.Reply to this email directly, view it on GitHub, or mute the thread.

**Comment by tuarua** [comment]
This is now fixed in 1.3.2

**Comment by lgignoud** [comment]
Fantastic support for this amazing piece of tech.
Will test it as soon as I can.
Thanks !
